netket/nn/symmetric_linear.py

```diff
@@ -678,7 +678,9 @@ def DenseSymm(symmetries, point_group=None, mode="auto", shape=None, **kwargs):
 
     Args:
         symmetries: A specification of the symmetry group. Can be given by a
-            nk.graph.Graph, a nk.utils.PermuationGroup, or an array [n_symm, n_sites]
+            :ref:`netket.graph.Graph`, a :ref:`netket.utils.group.PermuationGroup`, or an array
```

Comment:
Oh, and there was a typo we overlooked all this time. :sweat_smile: 

```suggestion
            :ref:`netket.graph.Graph`, a :ref:`netket.utils.group.PermutationGroup`, or an array
```